evm/15_core-concepts/49_common-erc-standards.md

```diff
@@ -0,0 +1,101 @@
+---
+title: EOAs vs Smart Contracts
+contributors:
+  - { name: Esteban Saá, github: stebansaa }
+---
+
+## Common ERC standards
+
+In the EOSEVM ecosystem, several widely adopted ERC (Ethereum Request for Comments) standards exist. These standards provide guidelines and specifications for implementing various functionalities and use cases on the EOSEVM network. Here are four commonly used ERC standards: ERC-20, ERC-721, ERC-777, and ERC-1155.
+
+### ERC-20
+ERC-20 is a standard interface for fungible tokens on the EOSEVM network. Fungible tokens are interchangeable and have identical values. ERC-20 tokens have methods to transfer tokens, check balances, and approve token spending on behalf of an address.
+
+#### Example ERC-20 Contract
+Here's an example of a basic ERC-20 token contract written in Solidity:
+
+```
```

Comment:
A full example of erc20 code like this really belongs in a guide specifically for erc20s.

If this is meant to be preliminary educational information, then the reader can't understand the code yet anyway. And if it's meant to be read as an advanced guide after they understand smart contract basics (which I think it should be) then it's unlikely anyone is creating an ERC20 like this anymore (and shouldn't, even if they are). It's better to import an openzeppelin contract as you did for the 721 that ensures security and cuts down the code the dev needs to write/understand.

Reply:
ok, will just remove it then.